modules/graphstudio/pages/explore-graph/actions.adoc

```diff
@@ -0,0 +1,37 @@
+= Actions
+
+The Actions tab in GraphStudio is a hub for actions that affect the entire graph database.
+
+== Clear all graph data
+
+image::clear-all-graph-data.png[]
+
+This command removes all loaded data from the schema while retaining all graph schemas and queries.
+
+Your user account must have the superuser role or have a role with the clear graph store privilege, and you must also be in Global View to clear all data from all graphs.
+You are required to type "clear all graph data" as a confirmation phrase to avoid accidentally performing this action.
+
+This operation will take about a minute depending on the size of your graphs and your hardware capability.
+
+[CAUTION]
+====
+Clear Graph Data deletes all data from your database and cannot be undone. Only the schema and queries will remain.
+
+If you clear the graph data by accident, you can start over and re-load the data into the database with the "start/resume loading" button image:start-loading-button.png[].
```

Comment:
I would still either remove the second paragraph or clarify that the data you can reload doesn't necessarily include all the data you lost. 

For example, I set up an application that can create entries into the database and use TigerGraph as the backend. Maybe I never made a loading job - all entries come in from REST requests. If I end up clearing graph data, none of the data will remain and I cannot run a loading job to recover that data, because I never had the data in a file.